mvp-spec/partials-updater/src/lexer.ts

```diff
@@ -1,6 +1,8 @@
-export const PartialBeginStartKey = "<!--partial-begin"
-export const PartialBeginEndKey = "-->\r\n"
-export const PartialEndKey = "<!--partial-end-->\r\n"
+import {Document} from "./files";
+
+export const PartialEnd = /^<!--partial-end-->[\s\S]*$/
+
+export const PartialBegin = /^<!--partial-begin *(.*) *-->[\s\S]*$/
```

Comment:
The regex `[\s\S]*$` includes also line breaks. As many as you have. I'm afraid it will match text we don't want to.